test/robot/ModelRegistry.py

```diff
@@ -10,7 +10,7 @@ def write_to_console(s):
 
 class ModelRegistry(mr.core.ModelRegistryAPIClient):
     def __init__(self, host: str = "localhost", port: int = 9090):
-        super().__init__(host, port)
+        super().__init__(host, port, "anything")
```

Comment:
thank you for the updates, but pardon me if I'm missing anything, per [previous comment](https://github.com/kubeflow/model-registry/pull/64#discussion_r1579746364) I'm not understanding why this is required.

Reply:
it's not, I forgot to take it out.